### ISSUE TITLE
Can I do runtime import from byte array?

### ISSUE THREAD

**Issue by marcusx2** [Siccity/GLTFUtility#190]
I don't want to pass a file as argument, but a byte array. The glb comes from the user's computer, where he taps on a button to show the files he wants to upload, then he selects the glb he wants. The glb is loaded to Unity in the form of a byte array.

**Comment by Unity3D-Hardik** [comment]
Yes that is possible , there are different method in "Importer.cs" , you can use ImportGLBAsync or LoadFromBytes method.
